experiments/integrated/ozark/ozark_parameters.jl

```diff
@@ -25,16 +25,23 @@ soil_vg_α = FT(0.04) # inverse meters
 z_0m_soil = FT(0.1)
 z_0b_soil = FT(0.1)
 soil_ϵ = FT(0.98)
-soil_α = FT(0.25)
+soil_α_PAR = FT(0.3)
+soil_α_NIR = FT(0.5)
 
-# Beer Lambert model parameters
+# TwoStreamModel parameters
 Ω = FT(0.69)
 ld = FT(0.5)
-ρ_leaf = FT(0.1)
-λ_γ = FT(5e-7)
+α_PAR_leaf = FT(0.1)
+λ_γ_PAR = FT(5e-7)
+λ_γ_NIR = FT(1.65e-6)
+τ_PAR_leaf = FT(0.05)
+α_NIR_leaf = FT(0.45)
+τ_NIR_leaf = FT(0.25)
+n_layers = UInt64(20)
+diff_perc = FT(0.2)
```

Comment:
The percentage of diffuse light can have a large impact on energy fluxes. We need to read that from a file or parameterize it. Does Ozark have incident diffuse radiation measurements?

Reply:
would it be ok to do this in a followup PR? adding the infrastructure to pass in diffuse fraction or fall back on an equation will require some more changes. this PR is already quite large.